### PR TITLE
[issue#12778][REF] sql_db: Add traceback stack if there is a  update in ir_config_parameter table

### PR DESCRIPTION
**Pull request by edy1192**

### CODE REVIEW

File: odoo/sql_db.py [hunk 1]
```diff
@@ -230,6 +236,15 @@ def execute(self, query, params=None, log_exceptions=None):
         try:
             params = params or None
             res = self._obj.execute(query, params)
+            if b'UPDATE' in self._obj.query.upper() and b'ir_config_parameter' in self._obj.query.upper():
```
Comment (moylop260) [inline odoo/sql_db.py:239]:
```suggestion
            if b'UPDATE' in self._obj.query.upper() and b'ir_config_parameter' in self._obj.query.lower():
```

Comment (edy1192) [reply odoo/sql_db.py]: Done. Thanks.

File: odoo/sql_db.py [hunk 2]
```diff
@@ -230,6 +236,15 @@ def execute(self, query, params=None, log_exceptions=None):
         try:
             params = params or None
             res = self._obj.execute(query, params)
+            if b'UPDATE' in self._obj.query.lower() and b'ir_config_parameter' in self._obj.query.lower():
```
Comment (moylop260) [inline odoo/sql_db.py:239]:
```suggestion
            if b'update' in self._obj.query.lower() and b'ir_config_parameter' in self._obj.query.lower():
```

Comment (edy1192) [reply odoo/sql_db.py]: Thanks. Done!

**Pull request closed without merging**